### PR TITLE
Add OIDC sinatra example app to dashboard

### PR DESCRIPTION
**Pull request by zachmargolis**
identity_dashboard

### CODE REVIEW

File: app/services/deploy_status_checker.rb [hunk 1]
```diff
@@ -47,6 +18,10 @@ def status_class
       end
     end
 
+    def short_name
+      app.gsub('identity-', '')
+    end
```
Comment (monfresh) [inline app/services/deploy_status_checker.rb]: Add a test for this?

Comment (zachmargolis) [inline app/services/deploy_status_checker.rb]: Done! eff7c088ae1259f299551b0af844a70ed805d017

File: config/status_checks.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    env: 'prod'
+    host: 'https://secure.login.gov'
+  - app: 'identity-sp-rails'
+    env: 'prod'
+  - app: 'identity-sp-sinatra'
+    env: 'prod'
+  - app: 'identity-dashboard'
+    env: 'prod'
+  - app: 'identity-openidconnect-sinatra'
+    env: 'prod'
```
Comment (zachmargolis) [inline config/status_checks.yml]: Configs without a host are not checked and shown as unavailable, so these are being "excluded" but we keep them in the UI for consistency (that's pre-existing behavior)

Comment (monfresh) [inline config/status_checks.yml]: Gotcha. Thanks for the explanation.

**Pull request merged**